### DISCUSSION TITLE
Raspberry Pi 5 Support

### DISCUSSION THREAD

**Discussion by BeerDiet** [mainsail-crew/mainsail#1718] [General]
I just got a RPi5 and wanted to upgrade from my 3B, and noticed that only RPi4 is supported.  I know the RPi4 can handle anything Mainsail and 3D printer people throw at it, but I got a great deal at Micro Center (only $15 extra for the 5).  Before I get into doing manual installs and builds, was just curious on the timeline that the 5 will be supported, if at all.  Thanks!

Edit:  Or should I just use KIAUH?

**Comment by meteyou** [comment]
I know this post is already closed, but I still want to write an answer here in case someone finds this post via Google or something similar.

We do not have Raspberry Pi 5 support yet, but we are already working on it. From the current point of view, the Raspberry Pi 5 is a worse choice than the Raspberry Pi 4 because it no longer has hardware video encoders for the webcam. Therefore, we still recommend the Raspberry Pi 4.

If someone still wants to use a Raspberry Pi 5 now, the easiest and fastest way is to flash a Raspberry Pi OS lite and then install the rest with KIAUH.

**Reply by BeerDiet** [reply]
Thanks for the response.  I closed it because I figured I'd just go the KIAUH route anyway for now, but this is really good to know and appreciated.  I did not realize the absence of the hardware video encoder, that's really interesting.
